### ISSUE TITLE
Cannot link account

### ISSUE THREAD

**Issue by ElementProHD** [kakopappa/sinric#130]
I keep on trying to connect my Alexa account and it won’t let me as it says failed to connect try again later

**Comment by kakopappa** [comment]
I assume you are referring to linking the Skill here

Can you access this page?
https://alexa.sinric.com/login

**Comment by ElementProHD** [comment]
Yes I can access this, what happens is a login and then receive the error message.

**Comment by kakopappa** [comment]
Can you send a snapshot

**Comment by ElementProHD** [comment]
“Unable to link your account”

**Comment by ElementProHD** [comment]
https://imgur.com/a/vDSXxF6

**Comment by kakopappa** [comment]
Sorry buddy I am not sure how to help you. I just tried linking and it works

![image](https://user-images.githubusercontent.com/7474406/46581569-f45bd700-ca64-11e8-9881-667929377e12.png)

Make sure your account is in one of the supported countries.

**Comment by ElementProHD** [comment]
Do you have any idea what to try ,, I made another Sinric account and tired it and same error? I’m using a iPhone do you think that could be the issue

**Comment by ElementProHD** [comment]
I have just tired on another device and still no luck

**Comment by ElementProHD** [comment]
It’s england and I have had it working about 6 weeks ago but not it suddenly doesn’t want to work, it was working fine before and responding perfectly on a previous project and I plugged that back in and was also disconnected from Alexa.

**Comment by kakopappa** [comment]
Not sure what I can do from this end. I think you have to contact support. You can do that from app.

**Comment by kakopappa** [comment]
Can you check now, please? I made few changes

**Comment by ElementProHD** [comment]
Hello I have resolved the issue it turns out that there is a bug on the Alexa App itself as I logged into several different accounts and had no luck linking them I then proceeded to logging in on my PC with 2 accounts and both successfully linked. The Alexa App was tired on IPhone X IOS 10 and IPhone 7 IOS 11
